dll/cpl/desk/lang/fr-FR.rc

```diff
@@ -122,11 +122,11 @@ BEGIN
             BS_AUTOCHECKBOX | WS_TABSTOP | WS_DISABLED, 10, 80, 267, 19
     CONTROL "&Afficher une ombre sous les menus", IDC_EFFAPPEARANCE_SETDROPSHADOW,
             "button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 95, 267, 19
-    CONTROL "Afficher le contenu des fenêtres pendant leur déplacement", IDC_EFFAPPEARANCE_DRAGFULLWINDOWS,
+    CONTROL "Afficher le contenu des fenêtres pendant leur déplacement", IDC_EFFAPPEARANCE_DRAGFULLWINDOWS,//FIXME: add accel
             "button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 110, 267, 19
-    CONTROL "Masquer les lettres soulignées pour la navigation au clavier jusqu'à ce que j'appuie sur la touche Alt",
+    CONTROL "Masquer les lettres soulignées pour la navigation au clavier jusqu'à ce que j'appuie sur la touche Alt",//FIXME: add back accelerator which has been destroyed during 0.4.15-dev'ing
             IDC_EFFAPPEARANCE_KEYBOARDCUES, "button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 125, 360, 19
-    CONTROL "Utiliser les menus plats", IDC_EFFAPPEARANCE_FLATMENUS,
+    CONTROL "Utiliser les menus plats", IDC_EFFAPPEARANCE_FLATMENUS,//FIXME: add back accelerator which has been destroyed during 0.4.15-dev'ing
```

Comment:
Note that this one is either an XP option, or a ReactOS-specific one, for activating an appearance that _does_ exist on Windows XP/2003 (and ROS)...
So I would suggest for example, putting the accelerator on the "p" of "plats".

![image](https://github.com/user-attachments/assets/7480394e-3db9-471c-8457-d688b92a46d7)

Reply:
Yes, I am aware, that this is a ReactOS-extra-feature, which Windows allows to access only via regedit. We can therefore use whatever accel we want to use here, which doesn't create a collision with an existing one. Sure, I can use "p" for french here, if that is your wish.

Reply:
This dialog is resolved now, and matches your french Windows screenshot to 100% regarding accels.